CI test fails: TimeoutError: Couldn't connect to the MCP server after 30 seconds

CI test fails:
> TimeoutError: Couldn't connect to the MCP server after 30 seconds

See: https://github.com/huggingface/smolagents/actions/runs/14795072325/job/41540256269?pr=1283
```
FAILED tests/test_tools.py::TestToolCollection::test_integration_from_mcp_with_sse - TimeoutError: Couldn't connect to the MCP server after 30 seconds
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -51,7 +51,7 @@ litellm = [
 ]
 mcp = [
   "mcpadapt>=0.0.19",  # Security fix
-  "mcp",
+  "mcp<1.7.0",  # Hotfix for GH-1284
 ]
 mlx-lm = [
   "mlx-lm"

